core/include/traccc/edm/internal_spacepoint.hpp

```diff
@@ -23,16 +23,7 @@ namespace traccc {
 template <typename spacepoint_t>
 struct internal_spacepoint {
 
-    // FIXME: geometry_id is hard-coded here
-    using spacepoint_container_type = host_container<geometry_id, spacepoint_t>;
-    using link_type = typename spacepoint_container_type::link_type;
-
-    link_type m_link;
-
-    /// TODO: This is a *VERY* temporary variable which will be removed asap.
-    /// only here while transitioning from the jagged to non-jagged
-    /// clusterization and prevent even more code duplication
-    unsigned int m_link_alt;
+    unsigned int m_link;
```

Comment:
I would prefer to keep the

```c++
using link_type = unsigned int;
```

declaration, and then using `link_type` in the code, sort of as before. Would reduce the amount of necessary changes a little.